packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1512,19 +1513,51 @@ module.exports = (
 
         const { uid, scope } = request.auth.credentials;
         const { ecosystemAnonId } = request.payload;
-        const noneMatchHeader = request.headers['If-None-Match'];
         const scopeSet = ScopeSet.fromArray(scope);
 
+        function parseMultiValueHeader(named) {
```

Comment:
Not in keeping with the HTTP spec, by contrast, is there any reason to let people submit more than one value for this header? I can't think of a scenario where a client would have more than one value for the old anon_id, and just having '*' or a single hash to evaluate would simplify the logic here.

Reply:
After talking this over, I think this was an unnecessarily paranoid suggestion :-)